BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -734,6 +736,7 @@ def main():
 	cmd_display.add_argument('-c',  dest='compress', choices=['lz4', 'lzma', 'dummy'],  default='dummy', help='compression algorithm')
 	cmd_display.add_argument('-a',  dest='auth', choices=['rsa2048', 'sha256', 'none'], default='none',  help='authentication algorithm')
 	cmd_display.add_argument('-k',  dest='key_file',  type=str, default='', help='Private key file path to sign component')
+	cmd_display.add_argument('-ht', dest='hash_type', type=str, required=True, help='Hash Alg for signing')
```

Comment:
-ht should be optional.   you should keep a default value if not specified.

Reply:
Sure, i would update to "SHA2_256" as default one.

Reply:
Can you give choices as well?

Reply:
Currently we have only SHA256 support. While adding SHA384 support in signing, I could add it.

Reply:
I am fine with it.